### DISCUSSION TITLE
generating website design

### DISCUSSION THREAD

**Discussion by suchavoda** [lllyasviel/Fooocus#2652] [Q&A] [answered]
Hi there!
I am trying to generate website preview, but nothing usable. Anybody here successfully use fooocus for website design? Thanks guys for advice!

**Comment by ILikeToasters** [comment]
You might want to look and see if any LoRAs for that. I have not really tried much of that sort of thing. I suppose it could give you ideas but not sure it would help with much more than that.

**Comment by bsdesigner** [comment, marked as answer]
Lora for Web Design: https://civitai.com/models/346885/web-design
Embedding: https://civitai.com/models/2502/webui-helper

**Reply by suchavoda** [reply]
This Lora helps. Thank you bsdesigner!
